modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -393,3 +393,48 @@ The Logging exporter prints data to the standard output.
           exporters: [logging]
 ----
 <1> Verbosity of the logging export: `detailed` or `normal` or `basic`. When set to `detailed`, pipeline data is verbosely logged. Defaults to `normal`.
+
+[id="prometheus-exporter_{context}"]
+==== Prometheus exporter
+
+The Prometheus exporter exports data using the Prometheus or OpenMetrics formats.
+
+* Support level: link:https://access.redhat.com/support/offerings/techpreview[Technology Preview]
+* Supported signals: metrics
+
+.OpenTelemetry Collector custom resource with an enabled Prometheus exporter
+[source,yaml]
+----
+  ports:
+  - name: promexporter <1>
+    port: 8889
+    protocol: TCP
+  config: |
+    exporters:
+      prometheus:
+        endpoint: 0.0.0.0:8889 <2>
+        tls: <3>
+          ca_file: ca.pem
+          cert_file: cert.pem
+          key_file: key.pem
+        namespace: prefix <4>
+        const_labels: <5>
+          label1: value1
+        enable_open_metrics: true <6>
+        resource_to_telemetry_conversion: <7>
+          enabled: true
+        metric_expiration: 180m <8>
+    service:
+      pipelines:
+        metrics:
+          exporters: [prometheus]
+----
+<1> Exposes the Prometheus port from the collector pod and service. You can enable scraping of metrics by Prometheus by using the port name in `ServiceMonitor` or `PodMonitor`.
+<2> The endpoint on which the metrics will be exposed.
```

Comment:
We don't use "will" in OCP docs: https://docs.engineering.redhat.com/display/CCSENABLE/Future+tense

```suggestion
<2>  The network endpoint where the metrics are exposed.
```